### DISCUSSION TITLE
allow multi-line text in chat-cli chat mode

### DISCUSSION THREAD

**Discussion by WPrintz** [chat-cli/chat-cli#44] [Ideas]
Currently chat-cli doesn't allow multi-line continuation in a chat session as far as I could determine.  Is it possible to allow using the \ character to escape the new line and let the prompt continue across multiple lines?  This would allow copy/pasting multiple lines of text if they are escaped properly.

**Comment by micahwalter** [comment]
That's a good callout. I think I can change the code to allow for multi-line.

**Comment by micahwalter** [comment]
I'm working on re-writing the chat module to be a TUI (Terminal User Interface) similar to what happens when you launch htop or vim. SO we will have multi-line mode like this https://github.com/charmbracelet/bubbletea/tree/main/examples/chat and features like ability to attach a file from your local machine, save chat, save configs, etc.
